scripts/libsimple_crontab

```diff
@@ -15,6 +15,8 @@ HOME=/var/www/circulation
 */15 * * * * root core/bin/run overdrive_monitor_full >> /var/log/cron.log 2>&1
 */15 * * * * root core/bin/run overdrive_monitor_recent >> /var/log/cron.log 2>&1
 */15 * * * * root core/bin/run overdrive_reaper >> /var/log/cron.log 2>&1
+0 2 * * * root core/bin/run enki_reaper >> /var/log/cron.log 2>&1
+0 0 0 */2 * root core/bin/run enki_monitor >> /var/log/cron.log 2>&1
```

Comment:
One more small change: day of the month starts at 1, not 0.
You can always check your work at that link I shared, https://crontab.guru/.